imagefit/models.py

```diff
@@ -16,9 +19,14 @@ class Image(object):
     Represents an Image file on the system.
     """
 
-    def __init__(self, path, cache=None, cached_name=None, *args, **kwargs):
+    def __init__(self, path, external=False, cache=None, cached_name=None, *args, **kwargs):
```

Comment:
What do you think if, instead of adding an external parameter, it would check if the `path` is an external url?
Something like `this.external = path.startswith(('http://', 'https://'))`

Reply:
thanks for the comment @vinyll